includes/multi-currency/class-currency.php

```diff
@@ -49,6 +51,14 @@ class Currency implements \JsonSerializable {
 	 */
 	private $rounding;
 
+	/**
+	 * Is currency zero decimal?
+	 *
+	 * @var bool|null
+	 */
+	private $is_zero_decimal;
```

Comment:
We could just initialise the value of `$is_zero_decimal` as `false` here, since the check in the constructor will only set it to `true` if it is zero decimal.

Reply:
Yep! I modified `is_default` too. Thanks for pointing it out! Good to remove an unneeded extra operation. Addressed in 9677db6

Reply:
Awesome! Thanks for making those changes. Let's :shipit:!